background.ts

```diff
@@ -23,6 +24,12 @@ let cfg: settings.Settings;
 let tray: Tray;
 let gone = false; // when true indicates app is shutting down
 let lastBuildDirectory = '';
+const singleInstanceLock = Electron.app.requestSingleInstanceLock();
+
+if (!singleInstanceLock) {
```

Comment:
We only use this once, might as well inline it:
```suggestion
if (!Electron.app.requestSingleInstanceLock()) {
  // We do not support multiple instances; focus the original one.
```